com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicCurves.cs

```diff
@@ -17,9 +17,9 @@ public class AlembicCurves : MonoBehaviour
         /// </summary>
         public Vector3[] Positions => positionsList.GetArray();
         /// <summary>
-        /// Returns an array where: the i-th entry returns how many points are contained in the i-th curve.
+        /// Returns an array where: the i-th entry returns the starting index of the i-th curve ( the i-th curve is between [StrideArray[i]..StrideArray[i+1]) ).
```

Comment:
Shouldn't that be: `the i-th curve is between [StrideArray[i]..StrideArray[i+1]-1)`

Reply:
I'd say curveOffsets or curvePointIndices speaks to me more than strideArray

Reply:
`curveOffsets` it is...